modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryServiceTest.java

```diff
@@ -640,6 +655,96 @@ public void testSearchObjectEntries() throws Exception {
 		_objectEntryLocalService.deleteObjectEntry(objectEntry2);
 	}
 
+	@Test
+	public void testValidateMaximumNumberOfObjectEntries() throws Exception {
+		_setUser(_guestUser);
+
+		HashMapDictionaryBuilder.HashMapDictionaryWrapper<String, Object>
+			objectConfigurationDictionary = HashMapDictionaryBuilder.create(
+				Collections.emptyMap());
+
+		HashMapDictionaryBuilder.put(
```

Comment:
Hey @thallescezar , can you put these values in the initialization of HashMapDictionaryBuilder? You can follow this [pattern](https://github.com/liferay/liferay-portal/blob/04aa57bf2c79629b8f3bff76e9102e882a8e01f7/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java#L289-L306), this way we avoid these many lines invoking put method.